README.md

```diff
@@ -81,11 +81,12 @@ Libraries for working with Erlang and OTP.
 - [midas-framework/beam](https://github.com/midas-framework/beam) - Interoperating with specific functionality in the Erlang runtime (BEAM).
 - [gleam-experiments/otp_code](https://github.com/gleam-experiments/otp_code) - Work with Erlang code at runtime.
 
-## Error handling
+## Error handling and logging
```

Comment:
Could you make a logging section instead please?

Reply:
done